src/activities/gletters/gletters.js

```diff
@@ -99,7 +99,7 @@ function initLevel() {
         // If "sublevels" length is not set in wordlist, use the words length
         maxSubLevel = level.words.length
     }
-    items.score.numberOfSubLevels = maxSubLevel;
+    items.score.numberOfSubLevels = maxSubLevel + 1;
```

Comment:
I agree with you, I am also confused by this but I have to switch my brain from software mode to user mode. A user expect the first level to be 1, not 0. And 5/5 means you at at the last level but not completed it.